frontend/src/container/SideNav/index.tsx

```diff
@@ -57,7 +58,7 @@ const SideNav = ({ toggleDarkMode }: Props): JSX.Element => {
 	return (
 		<Sider collapsible collapsed={collapsed} onCollapse={onCollapse} width={200}>
 			<ThemeSwitcherWrapper>
-				<ToggleButton checked={isDarkMode} onChange={toggleTheme} />
+				<ToggleButton checked={isDarkMode} onChange={toggleTheme} defaultChecked={true} />
```

Comment:
please check `defaultChecked` according to the darkMode

don't set the explicit value

Reply:
Ok

Reply:
@pal-sig  changed